src/main/js/controls/Graph/helpers/helpers.js

```diff
@@ -541,7 +541,7 @@ const scaleGraph = (scale, config) => {
     if (config.axis.x.rangeRounding) {
         scale.x.nice();
     }
-    if (config.axis.y.rangeRounding) {
+    if (config.allowCalibration && config.axis.y.rangeRounding) {
```

Comment:
@SharathAlur does he configuration of the y2 axis (line 556) also need this update to the conditional check?

Reply:
I had missed it. I fixed here https://github.com/cerner/carbon-graphs/pull/191/commits/781c04e595b4fd9273aad6e01e8b7b465abdb62c